marquez_airflow/dag.py

```diff
@@ -193,6 +206,62 @@ def handle_callback(self, *args, **kwargs):
 
         return super().handle_callback(*args, **kwargs)
 
+    def _alternate_create_flow(self, step, ti):
```

Comment:
It feels like splitting this method into `startRunFlow` (that creates the referred datasets, the job, the run and starts it) and `endRunFlow` (that updates the output datasets and complete/fails the run) would allow for reuse between the "alternate" and "original" flow.